layouts/case-studies/list.html

```diff
@@ -21,6 +21,11 @@
 		</div>
 	</div>
 </section>
+<style>
+    #video button:hover {
+        background-color: #303539;
```

Comment:
```suggestion
        background-color: #303030;
```

We can use a shade of Kubernetes-grey